exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/PageReader.java

```diff
@@ -445,25 +450,29 @@ public void clear(){
    * @throws IOException An IO related condition
    */
   void resetDefinitionLevelReader(int skipCount) throws IOException {
-    if (parentColumnReader.columnDescriptor.getMaxDefinitionLevel() != 0) {
-      throw new UnsupportedOperationException("Unsupoorted Operation");
-    }
+    Preconditions.checkState(parentColumnReader.columnDescriptor.getMaxDefinitionLevel() == 1);
```

Comment:
@sachouche Please review fix for DRILL-6447

Reply:
@vrozov,

I believe that max-definition can be either zero or one. Zero if all columns are null.

FYI - Did you also include the fix that I made in "VarLenBulkPageReader.java"

Reply:
In case `getMaxDefinitionLevel()` is zero, the `resetDefinitionLevelReader()` should not be called as it unconditionally reads `definitionLevels` that is only valid when `getMaxDefinitionLevel()` is not zero.

Reply:
Yes, I included the other fix as well. Please review changes to `VarLenBulkPageReader.java`.

Reply:
Thank you Vlad! 
LGTM